### PR TITLE
Added Bolt 1.1 platform config and motor mapping.

### PR DESCRIPTION
**Pull request by tobbeanton**
A finding made during motor output testing was that the expansion tests where not compiled in. Therefor these where added to Kbuild so the outputs could be tested.

### CODE REVIEW

File: src/drivers/src/motors_def_cf2.c [hunk 1]
```diff
@@ -581,3 +606,14 @@ const MotorPerifDef* motorMapBoltBrushless[NBR_OF_MOTORS] =
...
 
+/**
+ * Brushless motors mapped to the Bolt 1.1 PWM outputs.
+ */
+const MotorPerifDef* motorMapBolt11Brushless[NBR_OF_MOTORS] =
+{
+  &BOLT_M1_BL,
+  &BOLT_M2_BL,
+  &BOLT_M3_BL,
+  &BOLT11_M4_BL
```
Comment (knmcguire) [inline src/drivers/src/motors_def_cf2.c:617]: Why is this one called BOLT11_M4_BL while the other ones are called BOLT_M#_BL?

File: src/drivers/src/motors_def_cf2.c [hunk 2]
```diff
@@ -301,6 +301,31 @@ static const MotorPerifDef BOLT_M4_BL =
     .preloadConfig = TIM_OC4PreloadConfig,
 };
 
+// Bolt 1.1 M4, PB10, TIM2_CH3, Brushless config
```
Comment (knmcguire) [inline src/drivers/src/motors_def_cf2.c:304]: Hi is there only here a addition for motor M4?

**Review by knmcguire — COMMENTED** [review]
I still saw something a bit weird in the PR. Seems like only Motor M4 was added. 

And what are the differences in the motor defines between the bolt and bolt 1.1?

File: src/drivers/src/motors_def_cf2.c (hunk 2 above)
Comment (tobbeanton) [reply src/drivers/src/motors_def_cf2.c:304]: Yes, this is the only HW change.

File: src/drivers/src/motors_def_cf2.c (hunk 1 above)
Comment (tobbeanton) [reply src/drivers/src/motors_def_cf2.c:617]: We could decouple it from the platform and call it e.g. PA1_TIM2_CH2_BL and do the same with all of them.

**Review by knmcguire — APPROVED** [review]

**Pull request merged**